jigasi.yml

```diff
@@ -9,7 +9,6 @@ services:
             - '${JIGASI_PORT_MIN:-20000}-${JIGASI_PORT_MAX:-20050}:${JIGASI_PORT_MIN:-20000}-${JIGASI_PORT_MAX:-20050}/udp'
         volumes:
             - ${CONFIG}/jigasi:/config:Z
-            - ${CONFIG}/transcripts:/tmp/transcripts:Z
```

Comment:
I think this was added here so someone can use the custom properties files, to enable transcript storage in this folder and to be extracting them from the host.

Reply:
Yes, but this is only used for transcriptions I believe. I have left this mount in the transcriber.yml use of jigasi

Reply:
Yep, I missed the other part.